index.js

```diff
@@ -27,29 +30,96 @@ const urls = fs
   .split(/\r?\n/)
   .map((l) => `${l}`);
 
-const addonUUID = 'd56a5b99-51b6-4e83-ab23-796216679614';
+let addonUUID = 'd56a5b99-51b6-4e83-ab23-796216679614';
```

Comment:
could be a single line with a ternary operator

Reply:
Changed.